scripts/west_commands/runners/nrfjprog.py

```diff
@@ -46,7 +46,7 @@ def do_exec_op(self, op, force=False):
         # Translate the op
 
         families = {'NRF51_FAMILY': 'NRF51', 'NRF52_FAMILY': 'NRF52',
-                    'NRF53_FAMILY': 'NRF53', 'NRF91_FAMILY': 'NRF91'}
+                    'NRF53_FAMILY': 'NRF53', 'NRF54L_FAMILY': 'NRF54L', 'NRF91_FAMILY': 'NRF91'}
```

Comment:
```
                    'NRF53_FAMILY': 'NRF53', 'NRF54L_FAMILY': 'NRF54L',
                    'NRF91_FAMILY': 'NRF91'}
```
newline?

Reply:
@gmarull has a PR in the works that will fix this